admin/option_values.php

```diff
@@ -225,7 +229,7 @@ function init() {
         </div>
         <div class="row">
             <?php echo zen_draw_form('update_product_attributes', FILENAME_PRODUCTS_OPTIONS_VALUES, 'action=update_product', 'post', 'class="form-horizontal"'); ?>
-            <?php echo zen_draw_hidden_field('products_update_id', $_GET['products_update_id']); ?>
+            <?php echo zen_draw_hidden_field('products_update_id', (isset($_GET['products_update_id']) ? (int)$_GET['products_update_id'] : '0')); ?>

```

Comment:
`$_GET['products_update_id']` where is this ever set as a $_GET?

Reply:
None found in a current version of Zen Cart.  There is reference to setting `$_GET['products_update_id'] = '';` in a few places where a product is being copied, (attributes_controller and category_product_listing), otherwise it seems that all instances are transferred/referenced by posting values.  Is it possibly for some historical plugin or other operation to remain backwards compatible? 

I've only looked back as far as ZC 1.5.5 for comparison/review as of this writing.

Reply:
ya, I can't see where they're relevant.
Removing.